cfg/common.go

```diff
@@ -152,6 +155,43 @@ type Common struct {
 	Monitoring MonitoringConfig `yaml:"monitoring"`
 }
 
+// GetBackends returns the list of backends from common configuration
+func (common Common) GetBackends() []string {
+	backends := []string{}
+	for _, cluster := range common.BackendsByCluster {
+		backends = append(backends, cluster.Backends...)
+	}
+
+	if (len(common.Backends) > 0) && (len(backends) > 0) {
+		log.Fatal("duplicate backend definition in config -- exiting")
+	}
+
+	if len(common.Backends) > 0 {
+		return common.Backends
+	}
+	return backends
+}
+
+// ClusterOfBackend returns the cluster of a given backend address from common configuration
+func (common Common) ClusterOfBackend(address string) (string, error) {
+	for _, cluster := range common.BackendsByCluster {
+
+		for _, backend := range cluster.Backends {
+			if backend == address {
+				return cluster.Name, nil
+			}
+		}
+	}
+
+	for _, backend := range common.Backends {
+		if backend == address {
+			return "", nil
+		}
+	}
+
+	return "", errors.Errorf("Couldn't find cluster for '%s'", address)
```

Comment:
It is better not to have capitals in errors. See https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
I agree with you, but all the instances of "errors.Errorf" in the project start with capitals.